src/shared/locales/index.js

```diff
@@ -79,6 +83,10 @@ export function getLanguage (locale = 'en-US') {
     return 'pt-BR'
   }
 
+  if (locale.startsWith('ru')) {
```

Comment:
I think your forked repository code is out of date.

`src/shared/locales/index.js`

https://github.com/agalwood/Motrix/blob/b627b389855c2caee48a114da846690b1292dab6/src/shared/locales/index.js#L59-L75

`ru` does not need to add this if judgment.

Reply:
Please remove this useless judgment.

```javascript
if (locale.startsWith('ru')) {
  return 'ru'
}
```